CorsixTH/Lua/hospital.lua

```diff
@@ -2440,3 +2380,19 @@ end
 function Hospital:announceStaffLeave()
   -- Nothing to do, override in a derived class.
 end
+
+function Hospital:makeNoTreatmentRoomFax()
+  -- Nothing to do, override in a derived class.
+end
+
+function Hospital:makeNoDiagnosisRoomFax()
+  -- Nothing to do, override in a derived class.
+end
+
+function Hospital:makeEmergencyStartFax()
+  -- Nothing to do, override in a derived class.
+end
+
+function Hospital:makeEmergencyEndFax()
```

Comment:
I'm a bit concerned you don't have params here but you do in playerHospital. But it's probably me being sceptical and it's actually fine to do this.

Reply:
You're right, added to the new parts and the older stubs.